File: Swift/프로그래머스/0/181856. 배열 비교하기/배열 비교하기.swift
import Foundation

func solution(_ arr1:[Int], _ arr2:[Int]) -> Int {
    var arr1Score = arr1.count
    var arr2Score = arr2.count
    
    while true {
        if arr1Score > arr2Score {
            return 1
        } else if arr1Score < arr2Score {
            return -1
        } else {
            arr1Score = arr1.reduce(0, +)
            arr2Score = arr2.reduce(0, +)
        }

        if arr1Score == arr2Score {
            return 0
        }
    }
}

File: Python3/프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    arr1Score, arr2Score = len(arr1), len(arr2)
    
    while True:
        if arr1Score > arr2Score:
            return 1
        elif arr1Score < arr2Score:
            return -1
        else:
            arr1Score, arr2Score = sum(arr1), sum(arr2)
        
        if arr1Score == arr2Score:
            return 0
        